client/auth/api.js

```diff
@@ -2,18 +2,61 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-/*globals define*/
-
 define([
   'p-promise',
-  'client/lib/constants',
-  'client/lib/options',
-  'client/lib/url'
-], function (p, Constants, Options, Url) {
+  'client/lib/function',
```

Comment:
This module has been re-purposed. It was originally the base class for the lightbox and redirect APIs. I have since created a `baseUI` in `auth/base`. This module now exposes the public API for `fxaRelierClient.auth.*`